xwiki-platform-core/xwiki-platform-tag/xwiki-platform-tag-ui/src/main/resources/XWiki/TagCloud.xml

```diff
@@ -462,6 +462,7 @@ ol.tagCloud .ultraPopular {
 ## tagCloudSpace - the space to display the tag cloud for. If blank, display cloud for the whole wiki. Default is blank.
 #set ($tagCloudSpace = $xcontext.macro.params.get('space'))
 #set ($tagCloudSpaces = $xcontext.macro.params.get('spaces'))
+#set ($limit = $xcontext.macro.params.get('limit'))
```

Comment:
Small nitpick: I'd use `#set ($limit = $!xcontext.macro.params.get('limit'))` so that code using the variable doesn't need to remember to do `$!limit`